gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/cockpit/command/handler/DeleteEnvironmentCommandHandler.java

```diff
@@ -352,7 +348,6 @@ private void deleteEnvironment(ExecutionContext executionContext, EnvironmentEnt
         categoryRepository.deleteByEnvironmentId(environment.getId());
         dashboardRepository.deleteByReferenceIdAndReferenceType(environment.getId(), DashboardReferenceType.ENVIRONMENT);
         dictionaryRepository.deleteByEnvironmentId(environment.getId());
-        eventService.deleteOrUpdateEventsByEnvironment(environment.getId());
         scoringRulesetRepository.deleteByReferenceId(environment.getId(), ScoringRuleset.ReferenceType.ENVIRONMENT.name());
```

Comment:
If `delete` methods are only use here, we can remove from service and repository